common-rendering/src/components/liveBlockContainer.tsx

```diff
@@ -135,6 +135,12 @@ const LiveBlockContainer = ({
 					padding-left: ${LEFT_MARGIN_DESKTOP}px;
 				}
 				border-bottom: 1px solid ${neutral[86]};
+
+				@media (prefers-color-scheme: dark) {
+					background-color: ${neutral[10]};
+					color: ${neutral[100]};
+					border-bottom: 0;
+				}
```

Comment:
This component is also available for DCR, where dark mode isn't supported. We have the `darkModeCss` function in `common-rendering/lib.ts` for this:

```suggestion
				${darkModeCss(supportsDarkMode)`
					background-color: ${neutral[10]};
					color: ${neutral[100]};
					border-bottom: 0;
				`}
```

And `LiveBlockContainer` will take `supportsDarkMode: boolean` as a prop. See the `KeyEvents` component for an example.

**Aside:** I think the CSS vars solution may simplify this. If we use a CSS variable that has the same name in both light and dark mode, while dark mode remains unsupported in dotcom that variable will always be set to the light mode variant.